apps/prairielearn/src/ee/auth/lti13/lti13Auth.ts

```diff
@@ -220,7 +220,7 @@ const OIDCLaunchFlowSchema = z.object({
 //
 async function authenticate(req: Request, res: Response): Promise<any> {
   // https://www.imsglobal.org/spec/security/v1p0/#step-3-authentication-response
-  OIDCAuthResponseSchema.passthrough().parse(req.body);
+  z.looseObject(OIDCAuthResponseSchema).parse(req.body);
```

Comment:
This doesn't work as you expect, just tested, `z.looseObject` expects the shape as the argument.
```suggestion
  z.looseObject(OIDCAuthResponseSchema.shape).parse(req.body);
```
Same for other uses of looseObject. Alternatively, change the original schema to become a loose object schema, since this is only used once.

Reply:
Changed in [842439d](https://github.com/PrairieLearn/PrairieLearn/pull/12409/commits/842439de6261695a0cb2a1471e61424b92513b23) to the second option, make it a loose schema.